modules/op-performance-tuning-using-tektonconfig-cr.adoc

```diff
@@ -0,0 +1,63 @@
+// This module is included in the following assembly:
+//
+// *openshift_pipelines/customizing-configurations-in-the-tektonconfig-cr.adoc
+
+:_content-type: CONCEPT
+[id="op-performance-tuning-using-tektonconfig-cr_{context}"]
+= Performance tuning using TektonConfig CR
+
+You can modify the fields under the `.spec.performance` parameter in the `TektonConfig` custom resource (CR) to change high availability (HA) support and performance configurations for the {pipelines-shortname} controller.
+
+.Example: `TektonConfig` performance fields
+[source,yaml]
+----
+apiVersion: operator.tekton.dev/v1alpha1
+kind: TektonConfig
+metadata:
+  name: config
+spec:
+  performance:
+    disable-ha: false
+    buckets: 1
+    threads-per-controller: 2
+    kube-api-qps: 5.0
+    kube-api-burst: 10
+----
+
+The fields are optional. If you set them, the {pipelines-title} Operator includes most of the fields as arguments in the `tekton-pipelines-controller` deployment under the `tekton-pipelines-controller` container. The {pipelines-shortname} Operator also updates the `buckets` field in the `config-leader-election` configuration map under the `tekton-pipelines` namespace.
+
+If you do not specify the fields, the {pipelines-shortname} Operator applies the default values for the {pipelines-shortname} controller.
+
+[NOTE]
+====
+If you modify or remove any of the performance fields, the {pipelines-shortname} Operator updates the `tekton-pipelines-controller` deployment and the `config-leader-election` configuration map (if the `buckets` field changed), and recreates `tekton-pipelines-controller` pods.
+====
+
+The following are the modifiable fields:
+
+* `disable-ha`: Enable or disable the HA support. The default value for the pipeline controller is `false`.
```

Comment:
Why "for the pipeline controller"? And if the value of false means that HA is actually enabled by default, could you maybe explain this here?

Reply:
@mramendi I wrote "for the pipeline controller" to specify that the fields are optional. The default values therefore indicate what is the default configuration for the pipeline controller. Do you think the specification here is unnecessary?